Gardner.html

```diff
@@ -0,0 +1,172 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="Gardner.css">
+    <link href="https://fonts.googleapis.com/css2?family=Manrope:wght@500;700&display=swap" rel="stylesheet">
+    <script src="https://code.jquery.com/jquery-3.6.0.min.js"></script>
+</head>
+
+<body>
+    <nav class="navbar">
+        <!-- using max width so that it can be applicable on all the devices, -->
+        <div class="max-width">
+            <div class = "box">          
+                <div class="logo"><a href="#">Gardener</a></div>
+             </div>
+            <ul class="menu">
+                <li><a href="#home">Home</a></li>
+                <li><a href="#about">About</a></li>
+                <li><a href="#Contact">Contact</a></li>
+            </ul>
+            <div class="menu-btn">
+                <i class="fa fa-bars"></i>
+            </div>
+        </div>
+    </nav>
+
+
+    <!-- home section -->
+
+    <section class="home" id="home">
+        <div class="max-width">
+            <div class="home-content">
+                <div class="text-1">Welcome to <span>Gardener</span></div>
+                <div class="text-2">Landscaping Company</div>
+                <div class = "text-3"><p>Lorem ipsum, dolor sit amet consectetur adipisicing elit. Molestias reprehenderit vitae, ipsam optio blanditiis eum ex perspiciatis expedita odit quisquam aliquam beatae minima dolores eveniet rem, molestiae ut dolore natus nam voluptates! Alias expedita odio delectus corrupti voluptatem reiciendis neque!</p></div>
+                <a href="#">Contact Us </a>
+            </div>
+        </div>
+        <div><img src="/Users/lakshyagoel/Downloads/images/img-1.jpg" class = "homeimg"></div>
+    </section>
+
+    <!-- about section -->
+
+    <section class="about" id ="about">
+        <div class = "max-width">
+            <div class = "offer">Offer</div>
+            <div class = "whatoffer">What We Offer</div>
+            <div class = "about-content">
+                <div>
+                    <img src="/Users/lakshyagoel/Downloads/images/ico-1.png" style="border-radius: 50%; height: 60px;">
```

Comment:
1. No inline-styling
2. Wrong asset paths

Reply:
Will keep that in mind